protected/humhub/modules/user/services/AuthClientService.php

```diff
@@ -120,6 +119,16 @@ public function createRegistration(): ?Registration
             return null;
         }
 
+        if (
+            $this->authClient->getId() === 'linkedin' &&
```

Comment:
It should not be necessary to include module specific logic in the core.

Maybe you can use this example: to map the attributes on Module level.
https://github.com/humhub-contrib/auth-github/blob/master/authclient/GithubAuth.php#L30

Reply:
Ok

Reply:
But you have some problem with 'lastname' parameters. Look carefully PR

Reply:
@yuriimaz Yes thanks, I think we should merge this part.